go/cmd/ct-fetch/ct-fetch.go

```diff
@@ -520,12 +531,13 @@ func (ld *LogSyncEngine) NewLogWorker(ctx context.Context, ctLogData *types.CTLo
 
 func (lw *LogWorker) sleep(ctx context.Context) {
 	// Sleep for ctconfig.PollingDelay seconds (+/- 10%).
-	duration := time.Duration((1 + 0.1*rand.NormFloat64()) * float64(*ctconfig.PollingDelay))
```

Comment:
oof

Reply:
That one was just cleanup, I was multiplying by time.Second in line 535. The bad one was `evaluationTime` in the health check. That one was causing the ct-fetch process to be rebooted every 7 minutes or so...